platform.md

```diff
@@ -1045,7 +1143,9 @@ For more information on build reproducibility see [https://reproducible-builds.o
 # layer metadata
 
 [run-image]
+  image = "<image>"
```

Comment:
This might be removed in favor of expanding reference to include the tag part (cc @ jjbustamante)